team_production_system/serializers/availability.py

```diff
@@ -29,34 +35,28 @@ def create(self, validated_data):
         # Check if end time is between a start time and end time of
         # an existing availability
         overlapping_condition2 = Availability.objects.filter(
-            mentor=mentor,
-            start_time__lt=end_time,
-            end_time__gt=end_time
+            mentor=mentor, start_time__lt=end_time, end_time__gt=end_time
         ).exists()
         # Check if start time is between the new start_time and new end_time
         overlapping_condition3 = Availability.objects.filter(
-            mentor=mentor,
-            start_time__gte=start_time,
-            start_time__lt=end_time
+            mentor=mentor, start_time__gte=start_time, start_time__lt=end_time
         ).exists()
         # Check if end time is between the new start_time and new end_time
         overlapping_condition4 = Availability.objects.filter(
-            mentor=mentor,
-            end_time__gt=start_time,
-            end_time__lte=end_time
+            mentor=mentor, end_time__gt=start_time, end_time__lte=end_time
         ).exists()
 
         availability_overlap = (
             overlapping_condition1
```

Comment:
One comment for changing all these function condition arguments to be condition_x.